Project-Black/Assets/scripts/PlayerHealth.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class PlayerHealth : MonoBehaviour
+{
+    public static event Action PlayerDied;
+
+    [SerializeField]
+    private int maxLives = 3;
+    [SerializeField]
+    private int currentLives;
+    public Text livesShown;
+    private void Start()
+    {
+        currentLives = maxLives;
+        livesShown.text = currentLives.ToString();
+    }
+
+    public void TakeDamage(int amount)
+    {
+        currentLives -= amount;
+        livesShown.text = currentLives.ToString();
+        DeathCheck();
+    }
+
+    private void DeathCheck()
+    {
+       if(currentLives <= 0)
+        {
+            PlayerDied?.Invoke();
+        }
```

Comment:
Your code all looks good, although just for the sake of finding something I noticed that the brackets in this if statement are slightly off center haha. In general looks good so far.